internal/database/mysql.go

```diff
@@ -103,7 +103,10 @@ func (db *MySQLDatabase) AuthenticateLogin(username string, password string) (id
 	).Scan(&id, &expectedPassword)
 	if errors.Is(err, sql.ErrNoRows) {
 		return 0, errors.New("invalid username")
-	} else if expectedPassword != password {
+	} 
```

Comment:
fix whitespace, if you prefer, we can simply let this PR in, then i can do a separate PR just for reformatting all the files you've changed, but I would prefer if we could just kee clean formatting on all PRs

Reply:
I believe this this has been fixed.

Reply:
I believe this has been fixed.